editor/modes/visual-editor/format-controls.js

```diff
@@ -0,0 +1,105 @@
+/**
```

Comment:
Oh, I see now the `LinkControl` component is inlined in the file, but the default export is the array of formatting controls. Hmm...